ext/tiovx/gsttiovxisp.c

```diff
@@ -2421,30 +2436,43 @@ get_imx390_ae_dyn_params (IssAeDynamicParams * p_ae_dynPrms)
   return status;
 }
 
-static int32_t get_imx728_ae_dyn_params (IssAeDynamicParams *p_ae_dynPrms)
+IssAeDynamicParams imx728_ae_dynPrms;
```

Comment:
for caching.
`gsttiovxisp.c` calls `get_imx728_ae_dyn_params` every frame.
if we want to avoid reading the yaml from sd card/nfs every frame, we need to cache it globally the first time `get_imx728_ae_dyn_params` is called.
`linux_aewb_module.c` calls `get_imx728_ae_dyn_params` only once when the lib is initialized so we didn't have to cache.